### ISSUE TITLE
Grouping of rows not working as expected when sorted

### ISSUE THREAD

**Issue by ap20feb** [nicolaskruchten/pivottable#996]
Grouping of rows is not working properly when i sort the table. Surprisingly, same code works fine on Mozilla Firefox but not in Chrome.

I am attaching the screenshots
 
Here rows are not grouped in chrome
![image](https://user-images.githubusercontent.com/6327269/44466128-29d76d00-a63d-11e8-986f-2a8f6c085a21.png)

While it works perfectly on firefox
![image](https://user-images.githubusercontent.com/6327269/44466222-5d19fc00-a63d-11e8-9d33-31d93b103eb1.png)

Any idea why?

**Comment by ap20feb** [comment]
I agree sorting behaviour will be different for different system/browser. Shouldn't grouping of cell work properly? Also, can you suggest me how I can achieve the desired grouping.

**Comment by nicolaskruchten** [comment]
Grouping of cells is working properly in this case, because grouping is of adjacent cells with the same key. In this case the browser's sorting algorithm happens to not be placing certain cells adjacent to one another. To really solve this, you would need to include all the cell labels in the sorting algorithm as a "tie breaker" for when the actual value is the same, but this isn't something I can picture pushing into the core of the library unfortunately.
